hack/local-up-kubeedge.sh

```diff
@@ -141,7 +141,7 @@ build_edgecore
 
 kind_up_cluster
 
-export KUBECONFIG="$(kind get kubeconfig-path ${CLUSTER_CONTEXT})"
```

Comment:
Now in kind v0.7.0, it places the kubeconfig to `$HOME/.kube` and named it to config.